dev.py

```diff
@@ -48,34 +49,80 @@ def cmd_all_npm_build(args):
         run_verbose(["npm", "run", "build"], cwd=str(frontend_dir))
 
 
-def cmd_all_install_python_deps(args):
-    """"Install all dependencies needed to run e2e tests for all examples and templates"""
+def cmd_e2e_build_images(args):
+    """Build docker images for each component e2e tests"""
     for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
```

Comment:
Is it possible to aggregate all examples in one docker container? Setup docker takes some time and we probably only need separate dockers for template and template-reactless. Probably it can be done in later PR

Reply:
Yes, it's possible. Let's discuss it with Kamil tomorrow, but I like the idea :)

Reply:
Our goal is less than 15 minutes and we seem to be achieving this so there is no need to optimize. If I was optimizing something, I would put playwright in the docker image so that we don't have to install the browser multiple times, but that's not even needed now. Multiple images and multiple containers are not a problem for me as long as they are lightweight.

Reply:
Ok, fine for me for now. Do you agree @sfc-gh-pbelczyk ?